src/tools/skylark/java/com/google/devtools/skylark/skylint/DocstringUtils.java

```diff
@@ -274,6 +274,12 @@ private boolean nextLine() {
         line = "";
         return false;
       }
+      if (startOfLineOffset == 0 && docstring.charAt(startOfLineOffset) == '\n') {
+        line = "";
+        endOfLineOffset += 1;
+        return true;
+      }
+
```

Comment:
Why the new newline parse logic?

Reply:
also for missing one-line summary

Reply:
Ok, I see. Can you add a short comment right above the if, explaining that this catches the case where the first line is empty? That'll make it easier to follow this function's logic alongside the above and below comments.